pom.xml

```diff
@@ -599,131 +670,129 @@
                 </plugins>
             </build>
         </profile>
+        
         <profile>
             <id>coverage</id>
+            <activation>
+            	<property>
+            		<name>withCoverage</name>
```

Comment:
Any reason to use variables instead of profile name? Profile names are mandatory, and there are no much differences between `-P` and `-D`.

Reply:
yes, there are differences.
as I described in the issue, when you invoke one profile with -P, any other profile configured with activeByDefault will be deactivated. In this case you will need to pass all profiles. 
In the current POM, if you call mvn -Pcoverage, the run-its will be deactivated.

While you don't have this behavior with property. it is much more flexible when you need combine profiles.
btw, I removed the runITS from normal building just to facilitate install the plugin locally without have to wait all tests invoked to finish.